pages/veranstaltungen.js

```diff
@@ -40,6 +63,37 @@ export default function Events({ voteDoneArray }) {
           </StyledUl>
         </StyledSection>
       ))}
+
+      <StyledLink href={"/addevent"}>
+        <StyledIcon icon={faPlus} />
+      </StyledLink>
     </>
   );
 }
+
+const StyledSectionHeadlineAndButton = styled.section`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledIcon = styled(FontAwesomeIcon)`
+  color: white;
+  width: 1.5rem;
+  height: 1.5rem;
+`;
+
+const StyledLink = styled(Link)`
+  background-color: var(--secondary-color);
+  width: 2rem;
+  height: 2rem;
+  border-radius: 4px;
+  gap: 4rem;
+  padding: 1rem;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
```

Comment:
The styled link looks a bit dangerous, when you are setting it fixed. Do you think there is a more elegant way?

Reply:
hmmm... it's difficult to change this, because i want the button to stay in the left bottom corner :(. But i changed the display flex, justify content and align items :)! Thanks you!